package.json

```diff
@@ -36,10 +37,10 @@
     "eslint-plugin-jsdoc": "^24.0.0",
     "eslint-plugin-vue": "^6.2.2",
     "jest": "24.9.0",
-    "vue": "^2.6.11",
+    "typescript": "^3.8.3",
```

Comment:
I don't like `typescript` being required as a dependency. Looking forward to removing this.